man/J.Rd

```diff
@@ -2,57 +2,58 @@
 \alias{J}
 \alias{CJ}
 \alias{SJ}
-\title{ Creates a Join data table }
+\title{
+  Creates a join \code{data.table}
+}
 \description{
-  Creates a \code{data.table} to be passed in as the \code{i} to a \code{[.data.table} join.
+  Creates a \code{data.table} for use in \code{i} in a \code{[.data.table} join.
 }
 
 \usage{
-# DT[J(\dots)]                           # J() only for use inside DT[\dots].
-SJ(\dots)                                # DT[SJ(\dots)]
-CJ(\dots, sorted = TRUE, unique = FALSE)  # DT[CJ(\dots)]
+DT[J(\dots)]                              # J() only for use inside DT[...].
```

Comment:
now you see why this was commented before 🤪

R CMD check looks at this as a call to a function J, so it's throwing a block because there is no J function

(^this isn't quite right... error posted by Jan is best)